pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/advisor_helper.go

```diff
@@ -191,8 +192,18 @@ func (cra *cpuResourceAdvisor) getRegionMaxRequirement(regionName string) float6
 	}
 
 	res := 0.0
-	for _, numaID := range r.GetBindingNumas().ToSliceInt() {
-		res += float64(cra.numaAvailable[numaID])
+	switch r.Type() {
+	case types.QoSRegionTypeIsolation:
```

Comment:
I don't quite understand why only limits of main containers are considered here.